airbyte-webapp/src/locales/en.json

```diff
@@ -488,7 +488,7 @@
 
   "credits.credits": "Credits",
   "credits.whatAreCredits": "What are credits?",
-  "credits.buyCredits": "Buy credits",
+  "credits.buyCredits": "+ Buy credits",
```

Comment:
@edmundito - @timroes mentioned that it would be nice to change it to an svg icon, but in a different PR. If there are any updates to that, I am happy to make the change though!

Reply:
@edmundito I would keep this as a `+` for now, until we've redone the Button component and give it an option to specifcy an icon at which point I'd suggest we exchange all the existing `+` into real icons.